src/backend/access/transam/varsup.c

```diff
@@ -27,9 +27,6 @@
 #include "utils/syscache.h"
 
 
-/* Number of OIDs to prefetch (preallocate) per XLOG write */
```

Comment:
we should not move the define, rather we should add a wrapper function to access it